### PR TITLE
[CHANGELOG] Update to release 2018.1b1

### PR DESCRIPTION
**Pull request by macartur**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -3,6 +3,49 @@ Changelog
...
+- Add 'active' and 'enabled' flags to GenericEntity
+- Add 'enable'/'disable' methods to child GenericEntity classes.
+- Define available_tags according to link's interfaces.
+- Endpoint ('/ui/all') to display a json with all napps ui components.
+- Endpoint ('/ui/<path:filename>') to get file with a specific napp component.
+
+Changed
+=======
+- Moved Interface class to interface.py file
+- Added link attribute to the Interface class
```
Comment (renanrodrigo) [inline CHANGELOG.rst]: Added: should be above

File: CHANGELOG.rst [hunk 2]
```diff
@@ -3,6 +3,49 @@ Changelog
...
+- Small refactore of Switch class.
```
Comment (renanrodrigo) [inline CHANGELOG.rst]: s/refactore/refactor/g

File: CHANGELOG.rst [hunk 3]
```diff
@@ -3,6 +3,49 @@ Changelog
...
+========
+
+[2018.1b1] - "dalva" beta1 - 2018-3-09
+**************************************
+Added
+=====
+- Added some new blueprints
+- Added Interface metadata informations
+- The Kytos controller have a Link class
+- Add 'active' and 'enabled' flags to GenericEntity
```
Comment (renanrodrigo) [inline CHANGELOG.rst]: GenericEntity itself was added in this version also

**Review by renanrodrigo — CHANGES_REQUESTED** [review]

File: CHANGELOG.rst [hunk 4]
```diff
@@ -3,6 +3,50 @@ Changelog
...
+=====
+
+Security
+========
+
+[2018.1b1] - "dalva" beta1 - 2018-3-09
+**************************************
+Added
+=====
+- Added some new blueprints
```
Comment (beraldoleal) [inline CHANGELOG.rst]: Added some new blueprints (EP012, EP013 and EP014)

File: CHANGELOG.rst [hunk 5]
```diff
@@ -3,6 +3,50 @@ Changelog
...
+- Added Interface metadata informations
```
Comment (beraldoleal) [inline CHANGELOG.rst]: - Now, we have few Entities inside the core (Switch, Interface and Link)
- Each Entity has metadata attribute (a dict)

File: CHANGELOG.rst [hunk 6]
```diff
@@ -3,6 +3,50 @@ Changelog
...
+- Added link attribute to the Interface class
+- The Kytos controller have a Link class
```
Comment (beraldoleal) [inline CHANGELOG.rst]: remove this.

File: CHANGELOG.rst [hunk 7]
```diff
@@ -3,6 +3,50 @@ Changelog
...
+- GenericEntity itself was added in this version also
+- Add 'active' and 'enabled' flags to GenericEntity
```
Comment (beraldoleal) [inline CHANGELOG.rst]: Added 'active' and 'enable' flags to GenericEntity (EP013)

File: CHANGELOG.rst [hunk 8]
```diff
@@ -3,6 +3,50 @@ Changelog
...
+- Add 'enable'/'disable' methods to child GenericEntity classes.
+- Define available_tags according to link's interfaces.
+- Endpoint ('/ui/all') to display a json with all napps ui components.
+- Endpoint ('/ui/<path:filename>') to get file with a specific napp component.
+
```
Comment (beraldoleal) [inline CHANGELOG.rst]: - Now, kytosd is a python module, to make it easy to run with asyncio on the future;
- This pre-release implements EP013 and EP014 as discussed on our last Kytos Dev Meeting.

**Review by beraldoleal — CHANGES_REQUESTED** [review]

**Pull request merged**